external/vcm/vcm/calc/calc.py

```diff
@@ -55,3 +55,10 @@ def weighted_average(
         kwargs=kwargs,
         dask="allowed",
     )
+
+
+def vertical_scale_factors(n_levels: int, cutoff: int, rate: float):
```

Comment:
Adding "tapering" to the name might help make the purpose of this function clearer.
```suggestion
def vertical_tapering_scale_factors(n_levels: int, cutoff: int, rate: float):
```